src/devices/Ft4222/Ft4222Spi.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Ft4222;
+using System.IO;
+
+namespace System.Device.Spi
+{
+    /// <summary>
+    /// Create a SPI Device based on FT4222 chipset
+    /// </summary>
+    public class Ft4222Spi : SpiDevice
+    {
+        private readonly SpiConnectionSettings _settings;
+        private IntPtr _ftHandle = new IntPtr();
```

Comment:
if not too much work it would be beneficial to wrap this handle with SafeHandle (anywhere where it's used) - this make it much harder to leak

Reply:
Do you have a good example of SafeHandle? Something I can get the best practice out.